java/vector/src/main/java/org/apache/arrow/vector/complex/FixedSizeListVector.java

```diff
@@ -62,9 +62,6 @@ public static FixedSizeListVector empty(String name, int size, BufferAllocator a
 
```

Comment:
This class still uses `BitVector` as validity. If we are not changing this in this PR, maybe leaving a TODO?

Reply:
I am currently doing it.